ssv/value_check.go

```diff
@@ -99,6 +101,9 @@ func ProposerValueCheckF(
 		}
 
 		if blockData, _, err := cd.GetBlindedBlockData(); err == nil {
```

Comment:
isn't it better to run this code only if `supportsBlinded` is true?

Reply:
I think no, because if blinded is off and a blinded block is passed, it'd be good to return a special error (the one below) instead of something like "failed to decode ..."